Tools/TestWebKitAPI/Tests/WTF/ListHashSet.cpp

```diff
@@ -521,6 +521,45 @@ TEST(WTF_ListHashSet, UniquePtrKey_RemoveUsingRawPointer)
     EXPECT_EQ(1u, ConstructorDestructorCounter::destructionCount);
 }
 
+TEST(WTF_ListHashSet, EqualityComparison)
+{
+    ListHashSet<int> emptyList;
+
+    ListHashSet<int> nonEmptyList1;
+    nonEmptyList1.add(1);
+    nonEmptyList1.add(2);
+
+    ListHashSet<int> nonEmptyList2;
+    nonEmptyList2.add(2);
+    nonEmptyList2.add(3);
+
+    ListHashSet<int> nonEmptyList3;
+    nonEmptyList3.add(1);
+    nonEmptyList3.add(2);
+
+    ListHashSet<int> nonEmptyList4;
```

Comment:
Since operator== can have any container on the right hand side, maybe try testing it with a non-ListHashSet container.

Reply:
I'll add some test cases for that.